### PR TITLE
fix(excel): checking the excel for duplicate ids and fixing duplicates check

### PR DESCRIPTION
**Pull request by MayaSasson**

### CODE REVIEW

File: client/src/Utils/vendor/useDuplicateContactId.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import logger from 'logger/logger';
+import { Service, Severity } from 'models/Logger';
+import useCustomSwal from 'commons/CustomSwal/useCustomSwal';
+
+export const duplicateIdsErrorMsg = 'found duplicate ids';
```
Comment (sandymash) [inline client/src/Utils/vendor/useDuplicateContactId.ts:5]: the const is already exported in contactQuestioning, maybe delete it there?

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts [hunk 2]
```diff
@@ -337,7 +324,7 @@ const useContactQuestioning = (parameters: useContactQuestioningParameters): use
     };
 
     const checkForDuplicateIds = (idToCheck: string, interactedContactId: number) => {
-        return allContactedInteractions.findIndex((contact) => !idToCheck && contact.identificationNumber === idToCheck
```
Comment (daniel7764) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts:340]: why did you delete it? I think it's Orel's fix for the null ids bug

Comment (MayaSasson) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/useContactQuestioning.ts:340]: That makes the function to always return -1, the undefined check is also happening before calling the function

**Review by sandymash — APPROVED** [review]

**Review by daniel7764 — APPROVED** [review]
